README.md

```diff
@@ -66,6 +75,58 @@ $ make && bin/zotonic runtests mod_ginger_collection
 
 For more, see the [Docker](docs/docker.md) doc chapter.
 
+### 2. Selective Docker (recommended)
+
+Due to limitations in Docker for Mac, file synchronization performance suffers
+when you have large amounts of files (i.e. many sites with node_modules/ 
+directories).
+
+By running Zotonic directly on your host (outside Docker) we circumvent this
+limitation. Another advantage is that you can directly make changes in Zotonic
+source code, too. 
+
+All other services (PostgreSQL, Elasticsearch, Kibana) still run in
+containers.
+
+First, install Install Erlang locally:
+
+```bash
+$ brew install erlang@20
+$ brew link erlang@20 --force
+```
+
+Then clone both Ginger and Zotonic:
+
+```bash
+$ git clone https://github.com/driebit/ginger.git
+$ git clone https://github.com/zotonic/zotonic.git --branch 0.x 
+```
+
+Copy Ginger’s configuration file, which includes its dependencies:
+
+```bash
+cp ginger/config/zotonic.config ~/.zotonic/0/zotonic.config
```

Comment:
Create dir `~/.zotonic/0/` if missing? Assuming it is, since you've never started Zotonic